api/sql/tripSQL/initialize.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS Trips CASCADE;
+
+CREATE TABLE Trips (
+    id SERIAL PRIMARY KEY,
+    leader INT,
+    date DATETIME,
+    start INT,
+    destinations INT[] REFERENCES POI(id),
```

Comment:
Is this valid SQL? Since usually a foreign key is declared on a separate line (like the ones below) - (Actually not specifically sure how you would make an INT[] of foreign keys).

Reply:
You can declare foreign keys inline or on a separate line. I am not sure if having an array of foreign keys is best practice - after doing some research, it looks like a good practice might be to have a separate table that connects a trip to a destination. What are your thoughts on that?

Reply:
Ah, cool - didn't know you could do it all inline!

As for the array, yeah - looking into it, I don't think there's any way to enforce an array to contain keys (i.e. it'd just be a list of integers that we'd HOPE would link to other table entries, but we have no way to verify that without checking). It probably would just be better/neater/more proper to create a new table to store the relations between trip and destination that we can query. I went and added a new table to the Database Design document, really simple, just a small 2-column table. Check that over, let me know if you have any thoughts, and if it looks good we can implement that.

Reply:
Thank you so much for updating the doc, the new table looks great to me!